react/modules/module01/README.md

```diff
@@ -5,7 +5,7 @@
 ### What should be done:
 
 1. Create a separate branch for this task.
-2. Use CRA with template --typescript.
+2. Use CRA with template *--typescript*, or use Vite with the template *react-ts*
```

Comment:
As there are recomendation for no more using CRA maybe it worth to ask students to use framework?

Reply:
https://github.com/reactjs/reactjs.org/pull/5487#issuecomment-1409720741
Well, I can't say that it is a strict recommendation to not to use CRA at all. It still exists, and even turned into a "launcher" it will allow to create mere client-side SPAs. Also considering, that this course's first 5 modules will be focused on client-side SPA, I think that it is fine to let students choose between Vite and CRA

Reply:
Made changes